data/helpers/actionRouter.js

```diff
@@ -0,0 +1,82 @@
+const express = require('express');
+
+const Action = require('../helpers/actionModel');
+
+const actionRouter = express.Router();
+
+
+// ACTION CRUD
+
+// get insert update remove
+
+actionRouter.get('/', async(req, res) => {
+    try {
+        console.log(res);
+        // const action = await Action.get(req.query);
+        const action = await Action.get(req.params.id);
+        if(action) {
+            res.json(action);
+        } else {
+            res.status(404).json({ message: 'The action could not be found. '});
+        }
+    } catch (error) {
+        console.log(error);
+        res.status(500).json({ message: 'Error retrieving the actions' });
+    }
+});
+
+// insert?
+actionRouter.post('/',checkLength, async(req, res) => {
+    try {
+        const action = await Actions.insert(req.body);
+        const { project_id, description, notes } = req.body;
+        if(project_id && description.length  &&notes){
+            return res.status(200).json(newAction)
```

Comment:
I don't see where newAction is coming from, it should throw you and undefined error